### PR TITLE
[KSQL-12662] Add default window size in WindowingIntTest

### PR DESCRIPTION
**Pull request by VedarthConfluent**
### Description 
tldr; This PR fixes the ksql build by updating the usage of `TimeWindowedDeserializer`

The method used by our test has been deprecated for quite some time, it was recently removed in [this commit](https://github.com/confluentinc/ce-kafka/commit/8148361c090ef2749385004fb815f5d2caf6ab66) in ce-kafka, which broke our build. We have now used the more generic method and provided the window size as `Long.MAX_VALUE`, which is exactly what it was in the deprecated method

### Testing done 
Ran build and tests locally

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")
- [ ] Do these changes have compatibility implications for rollback? If so, ensure that the ksql [command version](https://github.com/confluentinc/ksql/blob/master/ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java#L41) is bumped.

### CODE REVIEW

**Review by trnguyencflt — APPROVED** [review]

**Comment by trnguyencflt** [comment]
It looks like we got related failed tests

**Review by hrishabhg — CHANGES_REQUESTED** [review]
Need context on PR.

File: ksqldb-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java [hunk 1]
```diff
@@ -68,8 +68,10 @@ public class WindowingIntTest {
 
   private static final StringDeserializer STRING_DESERIALIZER = new StringDeserializer();
 
+  private static final long DEFAULT_WINDOW_SIZE = Long.MAX_VALUE;
+
   private static final TimeWindowedDeserializer<String> TIME_WINDOWED_DESERIALIZER =
-      new TimeWindowedDeserializer<>(STRING_DESERIALIZER);
+      new TimeWindowedDeserializer<>(STRING_DESERIALIZER, DEFAULT_WINDOW_SIZE);
```
[the remark this answers is not in the document]
Comment (VedarthConfluent) [reply ksqldb-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java:74]: Updated PR desription

**Comment by VedarthConfluent** [comment]
The changes introduced in this PR seem to be stable. There are few test failures but they are not related to this change

**Comment by trnguyencflt** [comment]
> The changes introduced in this PR seem to be stable. There are few test failures but they are not related to this change

@VedarthConfluent ksql still needs to tests to pass to publish jars, shall we disable the tests to unblock the build process and file a jira ticket to fix those tests later?

File: ksqldb-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mjsax) [reply ksqldb-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java:74]: There is no "default" window size -- this must be the actual window size of the data we want to deserialize.

Thus, I think we should pass in `Duration.ofSeconds(10).toMillis()` here, and also update the `shouldAggregateTumblingWindow()` and `shouldAggregateHoppingWindow()` accordingly.

Just found this PR after I hit this issue locally...

[the remark this answers is not in the document]
Comment (VedarthConfluent) [reply ksqldb-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java]: Build succeeds with warnings when deprecated API is used, hence this was never detected.
Made the recommended changes

**Review by mjsax — APPROVED** [review]

File: ksqldb-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mjsax) [reply ksqldb-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java]: Thanks.

I think you should consider to enable failing the build, or it's a manual process to watch out for deprecation warnings and file ticket to move of deprecated APIs (there is always at least a deprecation time frame of 1 year before things are removed, so maybe manually observing w/o failing the build could be sufficient).

**Review by hrishabhg — APPROVED** [review]
